pyproject.toml

```diff
@@ -8,7 +8,7 @@ packages = [{ include = "qualibrate_composite" }]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.9,<3.12"
```

Comment:
I'm not sure about the version specification "^3.9, <3.12"
If works - fine

Reply:
So, approved?